extension.js

```diff
@@ -1,363 +1,344 @@
-const St = imports.gi.St;
-const Main = imports.ui.main;
-const GLib = imports.gi.GLib;
-const Mainloop = imports.mainloop;
-const Gio = imports.gi.Gio;
-const Gettext = imports.gettext;
-const ExtensionUtils = imports.misc.extensionUtils;
-const Me = ExtensionUtils.getCurrentExtension();
-const Domain = Gettext.domain(Me.metadata.uuid);
-const _ = Domain.gettext;
-const ngettext = Domain.ngettext;
-const ExtManager = Main.extensionManager;
-
-let gschema;
-var settings;
-
-var locations = {};
-
-let cancellable = null;
-let executeQueue = [];
-let resultEvent;
+import St from 'gi://St';
+import GLib from 'gi://GLib';
+import Gio from 'gi://Gio';
+import * as Main from 'resource:///org/gnome/shell/ui/main.js';
+import {Extension, gettext as _} from 'resource:///org/gnome/shell/extensions/extension.js';
 
 const POSITIONS = {
     0: 'left',
     1: 'center',
-    2: 'right'
+    2: 'right',
 };
 
-function init() {
-    ExtensionUtils.initTranslations(Me.metadata.uuid);
```

Comment:
Is there a way to enable the translations as well? It seems like it was completely removed or I'm looking in the wrong place.

Reply:
From https://gjs.guide/extensions/upgrading/gnome-shell-45.html#extensionutils  

> `initTranslations()` Consider this method deprecated. Only specify gettext-domain in metadata.json. GNOME Shell can automatically initiate the translation for you when it sees the gettext-domain key in metadata.json.  

I assume it should just work. Should be verified through testing, of course.